samples/managed-app-sample/createUiDefinition.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
Can you please add a README file in this sample folder and explain the concept?

Reply:
Actually this was purposely taken out. We will add this back once Maanasa publishes hers to link to the learn microsoft doc.